template_feed/Microsoft.DotNet.Common.ProjectTemplates.6.0/content/ClassLibrary-CSharp/.template.config/localize/en.templatestrings.cs.json

```diff
@@ -0,0 +1 @@
+﻿{}
```

Comment:
@jonfortescue  I didn't sport it from the beginning, but in fact this PR doubles all files and with invalid names.
![image](https://user-images.githubusercontent.com/56045248/122470657-ff61b600-cfc6-11eb-9623-2d57a908f56c.png)
I think we should we fix it first before we merge the PR. 

Which file do you expect to be stored in the repo? 
- `templatestrings.json`
- `templatestrings.en.json` 
- `templatestrings.[all 16 lang codes].json`

I can fix it tomorrow and remove the wrong files: now they are named in wrong order in the repo: `en.templatestrings.json`

Reply:
definitely remove these files before merging! agreed agreed. this will be fixed on the next arcade update.

Reply:
https://github.com/dotnet/templating/pull/3307 fixes templatestrings.json names and adds strings to be translated. Hopefully after that one is in, the generated files will be correct.